static-builds/netlify_build.py

```diff
@@ -29,24 +41,18 @@
     ])
     print(f"Fetching download URL from {download_url}")
     response = httpx.get(download_url, follow_redirects=True)
-
-    if response.status_code // 100 != 2:
-        try:
-            print(response.json())
-        except json.JSONDecodeError:
-            pass
-
-        response.raise_for_status()
+    raise_response(response)
 
     # The workflow is still pending, retry in a bit
     while response.status_code == 202:
         print(f"{response.json()['error']}. Retrying in 10 seconds.")
         time.sleep(10)
         response = httpx.get(download_url, follow_redirects=True)
 
+    raise_response(response)
     url = response.json()["url"]
     print(f"Downloading build from {url}")
-    zipped_content = httpx.get(url, follow_redirects=True)
+    zipped_content = httpx.get(url, follow_redirects=True, timeout=3 * 60)
```

Comment:
Do we want to wait 10 minutes here as well?

Reply:
The wait here is only for the download, so it should in theory only take a few seconds. 10 minutes might be a little over the top, but we've got plenty of build minutes to spare, so I don't mind either way.

Reply:
Very well.